bindings/pyroot/pythonizations/python/ROOT/pythonization/_roofit/_rooabsdata.py

```diff
@@ -0,0 +1,33 @@
+# Authors:
+# * Hinnerk C. Schmidt 02/2021
+# * Jonas Rembser 03/2021
+
+################################################################################
+# Copyright (C) 1995-2020, Rene Brun and Fons Rademakers.                      #
+# All rights reserved.                                                         #
+#                                                                              #
+# For the licensing terms see $ROOTSYS/LICENSE.                                #
+# For the list of contributors see $ROOTSYS/README/CREDITS.                    #
+################################################################################
+
+
+from ._utils import _getter
+
+
+class RooAbsData(object):
+    def plotOn(self, *args, **kwargs):
+        """
+        Docstring
```

Comment:
I would suggest to put more explanation here, e.g. the info that C++ command args from the doxygen page should work as keyword args.

Reply:
You might be able to reap some of the functionality that's powering (in the interpreter)
`.help RooAbsData.plotOn`